src/entities/Server/handler.ts

```diff
@@ -13,8 +13,7 @@ export async function initializeServices(
     }
   }
 
-  process.on('SIGTERM', async () => {
-    console.log(`SIGTERM received: ${chalk.yellow('shutting down services')}`)
+  const shutdown = async () => {
```

Comment:
I'm curious about what will happen if an exception is raised while stopping a service, will it terminate the process loosing all values in `services` values?